.travis/before-install-linux-javascript.sh

```diff
@@ -3,7 +3,7 @@
 set -euo pipefail
 
 sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
-sudo add-apt-repository ppa:fkrull/deadsnakes -y
-sudo add-apt-repository ppa:rwky/nodejs -y
 sudo apt-get update -qq
+curl -sL https://deb.nodesource.com/setup_0.12 | sudo -E bash -
```

Comment:
Just curious, but would these instructions work as well?

http://austinpray.com/ops/2015/09/20/change-travis-node-version.html

In particular, it seems like what we really want is to add to `.travis.yml`:

```
env:
  - TRAVIS_NODE_VERSION="0.12"
```